Clients/src/presentation/components/ProjectsOverview/index.tsx

```diff
@@ -0,0 +1,67 @@
+import "./index.css";
+import { FC } from "react";
+import { Box, Typography } from "@mui/material";
+import Grid from '@mui/material/Grid2';
+import Button from '@mui/material/Button';
+import ProjectCard from "./ProjectCard";
+
+const projects = [
+    {
+        id: 1,
+        name: "Marketing AI service",
+        description: "This project includes all the governance process status of the Chatbot AI project implemented in the  company.",
+        last_update: "23 January 2024",
+        owner_name: "Adam McLawn",
+        compliense_status: "85%",
+        controls_completed: "10/92",
+        requirements_completed: "7/49"
+    },
+    {
+        id: 2,
+        name: "Marketing AI service",
+        description: "This project includes all the governance process status of the Chatbot AI project implemented in the  company.",
+        last_update: "23 January 2024",
+        owner_name: "Adam McLawn",
+        compliense_status: "85%",
+        controls_completed: "10/92",
+        requirements_completed: "7/49"
+    },
+    {
+        id: 3,
+        name: "Marketing AI service",
+        description: "This project includes all the governance process status of the Chatbot AI project implemented in the  company.",
+        last_update: "23 January 2024",
+        owner_name: "Adam McLawn",
+        compliense_status: "85%",
+        controls_completed: "10/92",
+        requirements_completed: "7/49"
+    }
+];
+
+const ProjectsOverview: FC = () => {
```

Comment:
In here, please make your components dynamic and in situations like this that we're using mock data, pass the mock data as the default value for your prop.
So this could be something like this:
`const ProjectsOverview: FC = ({ data }) => {`